packages/vector_graphics_compiler/example/bars.svg

```diff
@@ -0,0 +1,6 @@
+<svg id="svg_build_00" xmlns="http://www.w3.org/2000/svg" width="48px" height="48px" >
+  <g if="group_1">
+    <path id="path_1" d="M 0,19.0 L 48.0, 19.0 L 48.0, 29.0 L 0, 29.0 Z " fill="#000000" />
+    <path id="path_2" d="M 0,34.0 L 48.0, 34.0 L 48.0, 44.0 L 0, 44.0 Z " fill="#000000" />
+  </g>
+</svg>
```

Comment:
newline at EOF.

Is there a source for this file?

Reply:
This is pulled from the framework

Reply:
Maybe just add a comment about the source ofit